bcfms/src/bcfms/ipa/pages/SubmitProject/steps/Step3_Type.vue

```diff
@@ -40,43 +40,47 @@ defineExpose({ isValid });
             :resolver="zodProjectTypeResolver"
             name="projectType"
         >
-            <LabelledInput
-                label="Project Type"
-                hint="Select type of project. If project type is not listed, select Other"
-                input-name="projectType"
-                :error-message="$form.projectType?.error?.message"
-                :required="true"
-            >
-                <ConceptSelect
-                    id="projectType"
-                    ref="projectTypeField"
-                    v-model="ipa.projectDetails.projectType"
-                    graph-slug="project_assessment"
-                    node-alias="project_type"
-                />
-            </LabelledInput>
+            <div class="formfield-margin-bottom">
```

Comment:
Can this be added to the LabelledInput so we don't have to apply it everywhere?
Then the `formfield-margin-bottom` could be defined locally in that component.